fail2ban/client/jailsreader.py

```diff
@@ -50,6 +50,7 @@ def jails(self):
 		return self.__jails
 
 	def read(self):
+		self.__jails = list()
```

Comment:
Sorry, my mistake - there are 2 bugs (2 commits), this test case ist for 1st only. 
The bug in jailsreader that you mean is interactive only, I should think a bit about a test case